knowledge_base/bundle_variables/databricks.yml

```diff
@@ -0,0 +1,24 @@
+bundle:
+  name: bundle_variables
+
+variables:
+  no_default:
+    description: This is a variable with no default value
+
+  with_default:
+    description: This is a variable with a default value
+    value: hello
+
+  with_default_in_targets:
+    description: This is a variable with its default value defined in the targets section
+
```

Comment:
Pedantic, I know, but can we please add periods to the end of the sentences on all of these descriptions?